setup.py

```diff
@@ -8,6 +8,8 @@
 
 deps = [
     'marionette-client == 0.9',
+    'mozinstall == 1.11',
+    'mozlog == 2.10',
```

Comment:
We won't. Running setup.py would raise conflicts. At least I could also add a `mozlog >= 2.7` if you want.